src/screens/Lending/EnterDepositAmount.js

```diff
@@ -256,10 +257,10 @@ const combinedMapStateToProps = (state: RootReducerState): $Shape<Props> => ({
 });
 
 const mapDispatchToProps = (dispatch: Dispatch): $Shape<Props> => ({
-  calculateLendingDepositTransactionEstimate: (
+  calculateLendingDepositTransactionEstimate: debounce((
```

Comment:
what is the default timeout for debounce?

Reply:
0, changed to 500.